staging/src/k8s.io/apiserver/pkg/admission/plugins.go

```diff
@@ -144,6 +145,9 @@ func (ps *Plugins) NewFromPlugins(pluginNames []string, configProvider ConfigPro
 			}
 		}
 	}
+	if len(pluginNames) != 0 {
+		glog.Infof("Loaded %d admission controller(s) successfully in the following order: %s.", len(pluginNames), strings.Join(pluginNames, ","))
```

Comment:
This is a great start, but it'd be even better if we printed out separately the mutating and validating plugins in their own list.

Reply:
Good idea

Reply:
Sure, I will implement it on monday.

Reply:
https://github.com/kubernetes/kubernetes/pull/61003